code/game/objects/items/slime_heart.dm

```diff
@@ -0,0 +1,28 @@
+/obj/item/slime_heart
+	name = "Slime Heart"
```

Comment:
Remove capitalization here, if it's capitalized BYOND treats it as a proper noun.